common/ux-window.cpp

```diff
@@ -483,7 +483,7 @@ namespace rs2
         bool do_200ms = every_200ms;
         if (_query_devices && do_200ms)
         {
-            _missing_device = _ctx.query_devices(RS2_PRODUCT_LINE_ANY).size() == 0;
+            _missing_device = _ctx.query_devices(RS2_PRODUCT_LINE_ANY_INTEL).size() == 0;
```

Comment:
This affects Viewer as well.

Reply:
In viewer we use refresh_devices() function that recognize the platform device.